test/PowerShellEditorServices.Test/Language/CompletionHandlerTests.cs

```diff
@@ -109,6 +111,7 @@ public async Task CompletesVariableInFile()
         public async Task CompletesAttributeValue()
         {
             (_, IEnumerable<CompletionItem> results) = await GetCompletionResultsAsync(CompleteAttributeValue.SourceDetails).ConfigureAwait(true);
+            Assert.Equal(3, results.Count());
```

Comment:
If _this_ fails in flaky test, we'll know that it's _not_ a sorting issue (which it _shouldn't_ be, because we already sort it, but this has been hell to figure out).